src/color.rs

```diff
@@ -30,38 +30,28 @@ pub enum ColorType {
     /// Pixel is 8-bit BGR with an alpha channel
     Bgra8,
 
-    /// Pixel is of unknown color type with the specified bits per pixel. This can apply to pixels
-    /// which are associated with an external palette. In that case, the pixel value is an index
-    /// into the palette.
-    Unknown(u8),
-
     #[doc(hidden)]
     __Nonexhaustive,
 }
 
-/// Returns the number of bits contained in a pixel of `ColorType` ```c```
-pub fn bits_per_pixel(c: ColorType) -> usize {
```

Comment:
Or you could leave it as:

```rust
fn bits_per_pixel(c: ColorType) -> u16 {
    u16::from(self.bytes_per_pixel()) * 8
}
```

It doesn't really hurt to have it, the `bytes_per_pixel` already commits to having a whole number of bytes without needing the removal of this method.